tests/unit/test_adapter.py

```diff
@@ -86,3 +88,15 @@ def test_get_table_type(self):
             connection = adapter.acquire_connection("dummy")
             connection.handle  # trigger lazy-load
             self.assertEqual(adapter.get_table_type(target_relation), "iceberg_table")
+
+    def test_create_csv_table_slices_big_datasets(self):
```

Comment:
Can we add another test to use `custom_glue_code_for_dbt_adapter` with a big seed to verify that we do not make breaking change from the previous version?

Reply:
Sorry, I don't get what you mean. What would be the test scenario?

Reply:
I mean, the test case with custom_glue_code_for_dbt_adapter.
Since you are modifying how custom_glue_code_for_dbt_adapter is used, we need to test that portion too.

Your PR is not a tiny PR, and it can potentially impact all the existing workloads.
We need to verify that this does not impact the existing workload.

Reply:
Ah it was my bad, this test case already covered required one.